TrackletAlgorithm/AllStubMemory.h

```diff
@@ -186,6 +186,27 @@ class AllStub : public AllStubBase<ASType>
     data_.range(kASBendMSB,kASBendLSB) = bend;
   }
 
+  std::string getBitStr() {
```

Comment:
@carriganm95 Each of these `getBitStr` methods should be made `const`, i.e.,
`std::string getBitStr()` → `std::string getBitStr() const`